packages/backend/src/apps/frappe/common/get-current-user.js

```diff
@@ -0,0 +1,12 @@
+import { OAUTH_ENDPOINTS } from "./constants.js";
+import { getFrappeSiteURL } from "./utils.js";
+
+const getCurrentUser = async ($) => {
+    const siteUrl = getFrappeSiteURL($);
+    const response = await $.http.get(`${siteUrl}${OAUTH_ENDPOINTS.OPENID_PROFILE}`);
```

Comment:
Could you please follow the conventions and update this file?
Take a look at eg. `/workspace/packages/backend/src/apps/gitea/common/get-current-user.js`

Reply:
Done.